docs/gen-ai/gen-ai-spans.md

```diff
@@ -388,6 +401,20 @@ Function: A tool executed on the client-side, where the agent generates paramete
   Client-side operations are actions taken on the user's end or within the client application.
 Datastore: A tool used by the agent to access and query structured or unstructured external data for retrieval-augmented tasks or knowledge updates.
 
+**[4] `gen_ai.tool.call.arguments`:** > [!WARNING]
+> This attribute may contain sensitive information.
+
+It's expected to be an object - in case a serialized string is available
+to the instrumentation, the instrumentation SHOULD do the best effort to
+deserialize it to an object.
+
+**[5] `gen_ai.tool.call.result`:** > [!WARNING]
+> This attribute may contain sensitive information.
```

Comment:
I think all 3 new attributes should be mentioned in the section "Capturing instructions, inputs, and outputs"

Reply:
I think at the very least there should be a follow up issue about this.

Reply:
@alexmojaki : Sure, I can do that. Let me take a note of it for Follow-up.